lib/fast_gettext/translation.rb

```diff
@@ -72,6 +76,14 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+    alias :nsgettext :ns_
+
+    def np_(*args, &block)
+      context, key = args[0..1]
+      nargs = ["#{context}#{CONTEXT_SEPARATOR}#{key}"] + args[2..-1]
```

Comment:
CONTEXT_SEPARATOR does not exist

Reply:
hmm also this breaks caching since now s and p use different caches ?

Reply:
commit to define CONTEXT_SEPARATER was just merged.

s and p do use different caches and can have different values. I'm not sure if that's by design but that's how ruby-gettext works.

```
msgctxt "Printer"
msgid "Open"
msgstr "translation-one"

msgid "Printer|Open"
msgstr "translation-two"
```